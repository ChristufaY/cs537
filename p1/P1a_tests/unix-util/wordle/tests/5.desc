String test: play with the large wordlist
