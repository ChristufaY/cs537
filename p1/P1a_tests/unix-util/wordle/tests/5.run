./wordle ./tests/wordlist.txt abcdeiount
