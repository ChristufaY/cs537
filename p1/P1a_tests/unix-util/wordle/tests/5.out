ghyll
glyph
gypsy
lymph
myrrh
pygmy
shyly
sylph
wryly
